javascript/README.md

```diff
@@ -505,3 +505,32 @@ functionThatReturnsAPromise()
         // other code here!
     });
 ```
+
+## Module Definition
+
+When building a module that can be consumed by a number of different sources, we prefer
```

Comment:
Nah - if it's just something your consuming that will only ever live in one project, it doesn't need to be UMD.